src/mixtures/mixturemodel.jl

```diff
@@ -500,6 +500,27 @@ componentwise_logpdf(d::MultivariateMixture, x::AbstractVector) = componentwise_
 componentwise_logpdf(d::MultivariateMixture, x::AbstractMatrix) = componentwise_logpdf!(Matrix{eltype(x)}(undef, size(x,2), ncomponents(d)), d, x)
 
 
+function quantile(d::UnivariateMixture{Continuous}, p::Real)
+    ps = probs(d)
+    min_q, max_q = extrema(quantile(component(d, i), p) for (i, pi) in enumerate(ps) if pi > 0)
+
+    if iszero(p)
+        return(min_q)
+    elseif isone(p)
+        return(max_q)
+    end
+
+    tol =  Base.rtoldefault(typeof(p)) * abs(p)
```

Comment:
The tolerance refers to the support of the distribution, not the quantile level (see https://github.com/JuliaStats/Distributions.jl/blob/d6de7214e6d84df77d1eb3ec7a98d23862d1232d/src/quantilealgs.jl#L10). I.e., it should be based on `min_q` and `max_q` (e.g. `max(rtoldefault(typeof(min_q)),rtoldefault(typeof(max_q)))` as in Base).

If one wants to use a type-dependent relative tolerance (which IMO seems reasonable since it will give more sensible tolerances for e.g. Float32), it would be even better if it is not converted to an absolute tolerance here - the values of the initial bracket might be ridiculously large which would lead to an unnecessarily large tolerance. Instead the comparison in https://github.com/JuliaStats/Distributions.jl/blob/d6de7214e6d84df77d1eb3ec7a98d23862d1232d/src/quantilealgs.jl#L10 could be based on the scale of `lx` and `rx`.